weldx/asdf/util.py

```diff
@@ -542,6 +553,82 @@ def _get_instance_shape(
     return None
 
 
+class _ProtectedViewDict(MutableMapping):
+    def __init__(self, protected_keys, data=None):
+        super(_ProtectedViewDict, self).__init__()
+        self.__data = data
```

Comment:
nice try, but I'm pretty sure as long as something is called anything like `*data` @vhirtham will always find it 😉